tools/data-api/go.mod

```diff
@@ -36,3 +39,5 @@ require (
 )
 
 replace github.com/hashicorp/pandora/tools/sdk => ../sdk
```

Comment:
note this is still used since `testhelpers` (and `config` need somewhere else to live, but that's a cross-project thing)